src/applications/caregivers/actions/fetchFacilities.js

```diff
@@ -3,23 +3,68 @@ import environment from 'platform/utilities/environment';
 import { apiRequest } from 'platform/utilities/api';
 import content from '../locales/en/content.json';
 
-export const fetchFacilities = async ({
+const joinAddressParts = (...parts) => {
+  return parts.filter(part => part != null).join(', ');
```

Comment:
I updated it to be != due to `!== `null only strips out null values and not undefined values (because it compares them strictly). `!=` null strips out both null and undefined (because compares them more loosely and treats them as equivalent). Is using lodash a standard that is fine? I hesitate to bring in extra dependencies unless absolutely necessary.